pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/SegmentDeletionManager.java

```diff
@@ -54,21 +56,22 @@
   private static final long MAX_DELETION_DELAY_SECONDS = 300L;  // Maximum of 5 minutes back-off to retry the deletion
   private static final long DEFAULT_DELETION_DELAY_SECONDS = 2L;
   private static final String DELETED_SEGMENTS = "Deleted_Segments";
+  private static final String RETENTION_MS_SEPARATOR = "__RETENTION_MS__";
```

Comment:
Can you add an example here (in comments) of what the file name would look like? Does it have the absolute time at which the file needs to go away, or some time period that needs to be compared with the file creation time?  Why not encode the absolute time in a human readable format like "20220311T2300Z"?

Reply:
both works. currently it is encoding as a MS number at the back (e.g. if retention = 1d, then the file name will be `SOME_SEGMENT__RETENTION_MS__86400`. 

but we can also encode the absolute timestamp that the file will go away like you suggested. did you have preference on one over the other?